Description: Set additionally LC_MESSAGES=C and LC_ALL=C to
 circumvent libgcj10 exception with locale de_AT.UTF-8.
 This patch is a workaround to an ArrayIndexOutOfBoundsException evoked in
 java.text.SimpleDateFormat.formatWithAttribute (libgcj10, Version 4.4.2-4).
 The exception occures only between April and December and with the de_AT-UTF.8 locale.
Author: Johann Felix Soden <johfel@gmx.de>
Bug-Debian: http://bugs.debian.org/560594


--- a/pdftk/pdftk.cc
+++ b/pdftk/pdftk.cc
@@ -2896,6 +2896,11 @@
 	*/
 	static char my_lang[]="LANG=C";
 	putenv(my_lang);
+	static char my_lc_messages[]="LC_MESSAGES=C";
+	putenv(my_lc_messages);
+	static char my_lc_all[]="LC_ALL=C";
+	putenv(my_lc_all);
+
 
 	for( int ii= 1; ii< argc; ++ii ) {
 		version_b=
